src/supervisor/models.py

```diff
@@ -2,12 +2,11 @@
 This module holds the classes used by supervisor
 """
 
-import logging
 import os
 import sys
 import traceback
 
-from .message_handlers import ConsoleHandler, MessageHandler
+from .message_handlers import MessageHandler
 
 
 class Supervisor:
```

Comment:
**thought**: having to import supervisor from a module `from supervisor.models import Supervisor` seems unnecessary to me since it's the main module. 

I think `from supervisor import Supervisor` is expected.

Reply:
Please check my understanding on this. This could entail moving the `Supervisor` class into a new `supervisor.py` module, which would make the import become from `supervisor import supervisor` and the constructor become `sup = supervisor.Supervisor()`... but that doesn't get to `from supervisor import Supervisor`. So that's not the way to do it.

I suppose in `__init__.py` I could do `from .models import Supervisor` so that a client could call `from supervisor import Supervisor`. They could still call `from supervisor.models import Supervisor`, but that could just not be mentioned in the docs and they wouldn't know unless they looked in the source code. This still feels a little cheaty, though.